src/LICM.cpp

```diff
@@ -338,6 +338,10 @@ class GroupLoopInvariants : public IRMutator {
     };
 
     int expr_depth(const Expr &e) {
+        // We want to keep constants outermost
+        if (is_const(e)) {
+            return std::numeric_limits<int>::max();
```

Comment:
Is there any chance that a caller could increment the result ("I want it outside this other thing") and overflow?

Reply:
No, this is only used immediately below to sort and isn't exposed out of this translation unit.